src/components/layoutcomponents/BaseHead.astro

```diff
@@ -96,35 +88,34 @@ const {
 
 <!-- Canonical URL -->
 {
-  urls && urls.length > 0 ? (
-    urls.map((post, index) => (
+  !isHomePage &&
```

Comment:
Bit curious what the problem was with the <i18nHead/> option for the i18n sanity plugin.. This solution works fine and is a lot more configurable so it's a good solution